task/upload.go

```diff
@@ -431,6 +457,7 @@ func outputLocations(outURL string, relativePath string) ([]OutputName, []client
 				Outputs: &clients.OutputsRequest{
 					SourceSegments:     sourceSegments,
 					TranscodedSegments: true,
+					AutoMP4:            autoMp4s,
```

Comment:
super nit: We should probably rename this output (`OutputNameOSPlaylistHLS`) as it's not HLS anymore. I couldn't come up with a better name other than "OutputNameMain" which I don't like either, so not a priority. To keep in mind tho